test/unit/Controllers/UserController.test.ts

```diff
@@ -134,4 +134,36 @@ describe("UserController", function () {
             }
         });
     });
+
+    describe('GET /user/account', () => {
+        const userServiceStub = sinon.createStubInstance(UserService);
+        // @ts-ignore
+        userServiceStub.fetchUserAccount.resolves({email: 'email@test.com', password: password});
+
+        it('should update email and password of the user', async function () {
+            try {
+                const userController = new UserController(
+                    userServiceStub as unknown as UserService);
+                const response = {} as Response;
+                response.locals = {userId: 5};
+                response.json = sinon.spy((result) => {
+                    expect(result.password).to.be.equal(password);
+                    expect(result.email).to.be.equal('email@test.com');
+                }) as any;
+
+                response.status = sinon.spy((result) => {
+                    expect(result).to.equal(200)
+                    return response;
+                }) as any;
+
+                await userController.fetchUserAccount({
+                    body: {}
+                } as Request, response)
+            } catch (err) {
+                console.log('aaaaa', err)
```

Comment:
bbbbbb :D
I thought the logger is now working so you should have gotten here error in case you were testing that

Reply:
Overlooked it and trusted the linter which only looks at ```src/```.